### PR TITLE
Handle possible None enum value

### PR DESCRIPTION
**Pull request by kissgyorgy**
`default_value` can be None in `InputValueDefinitionNode` (and any other subclass
of `DefinitionNode.default_value` property), so it should be handled at all times.

Refactored the `enum_values_in_input_type` and `enum_values_in_field_args` to use
the same helper function, so they can consistently handle the same edge case.

### CODE REVIEW

**Comment by rafalp** [comment]
Hi! Can you add tests demonstrating the issue fixed here?

File: ariadne/enums.py [hunk 1]
```diff
@@ -154,23 +154,27 @@ def enum_values_in_object_type(
         yield from enum_values_in_field_args(field_name, field)
 
 
-@enum_values_in_types.register(GraphQLInputObjectType)
-def enum_values_in_input_type(
-    type_: GraphQLInputObjectType,
-    field_name,
-) -> Generator[InputFieldWithKeys, None, None]:
-    for input_name, field in type_.fields.items():
+def _get_field_with_keys(field_name, fields):
```
Comment (rafalp) [inline ariadne/enums.py:157]: We are ordering logic in our code from top to bottom, so this should be declared below `enum_values_in_field_args`.

**Review by rafalp — CHANGES_REQUESTED** [review]

**Comment by kissgyorgy** [comment]
> Hi! Can you add tests demonstrating the issue fixed here?

Yes, sorry. I will add a minimal reproduction about the issue.

**Comment by rafalp** [comment]
I'll tangentially schedule this on next release. Sounds like helpful thing to fix.

**Review by rafalp — APPROVED** [review]
